scripts/generate-code

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd "$(dirname "$0")/.."
+
+set -e
+./generate_grpc.sh
+
+# remove the code that was copied from the instance manager for integration
+# testing - we don't want to track it in this repo
+for d in "../longhorn-instance-manager/integration/rpc"/* ; do
+  rm -rf "integration/rpc/$(basename "$d")"
+done
```

Comment:
make generate-code  sounds like an excellent approach!
I see the update inside longhorn-engine https://github.com/longhorn/longhorn-engine/pull/934/files#diff-25f37c7e6dca244ab51e2726908cd9290102273ba49923a651e2570c050ec51eR1
Could we also build the make generate-code  for longhorn-instance-manager?